backend/pokemon/helpers.py

```diff
@@ -46,3 +46,9 @@ def get_or_create_pokemon(pokemon_data):
             new_pokemon = create_pokemon(pokemon)
             pokemons.append(new_pokemon)
     return pokemons
+
+
+def has_repeated_positions(pokemon_position):
+    for i in pokemon_position:
+        repeated_position = pokemon_position.count(i)
+        return repeated_position > 1
```

Comment:
This logic is a bit strange to me.
An easy way to check for repeated positions is checking if the set is different from the list.
E.g.

```python
def has_repeated_positions(pokemon_position_list):
    return pokemon_position_list != set(pokemon_position_list)
```

Also note I changed the parameter to `pokemon_position_list`. Change the variable name as well to it.